frontend/pages/[resourceType].tsx

```diff
@@ -67,8 +49,24 @@ export default function ResourceList({ ids, resourceType }: InferGetServerSidePr
             marginRight: '150px'
           }}
         >
-          {idItems.length > 0 ? ( //if items exist
-            <ul>{idItems}</ul>
+          {resourceInfo.length > 0 ? ( // if items exist
+            <div>
+              <Stack style={{ marginTop: '10px', marginBottom: '10px', marginLeft: '50px', marginRight: '50px' }}>
+                {resourceInfo.map(res => {
+                  return (
+                    res && (
+                      <div
+                        onClick={() => {
+                          setSelectedResource(res.id);
```

Comment:
Can we add a check here that sees if the selected resource is the one clicked on, and sets `selectedResource` to null if so? That way we could collapse resources with a second click